meetings/2020-08-11.md

```diff
@@ -3,27 +3,83 @@
+    - Current is developers decide if something is a bug fix or if it is an enhancement
+    - Each issue should be labeled as a bug/enhancement
+      - Once the PR comes in the PR contributor and the reviewer need to check that it is still a bug fix or feature enhancement
+    - Once the PR is merged into Dev, there will be another PR cherry-picking into the LTS branch for bugs
+      - The bug fix PR contributor is responsible for doing the cherry-pick and opening a new PR onto the LTS branch
+      - Update to contributing document to outline this process for contributors
+    - Do we want to back port all bug fixes?
+      - The more bug fixes we back port to the LTS version, the more likely we introduce new bugs into it
+      - We can label "critical bugs" that get back ported, all other bugs don't get back ported
+      - Right now we don't need to do this, just back port all bug fixes
+      - You lose some benefits of the LTS model this way
+  - Release cadence and process
+    - Current process
+      - 4 feature releases per year (roughly January, April, July, September)
+        - Only happens if there is a new enhancement
+      - Release candidate 1 month before feature release
+      - USGS does 4 support sprints each year between feature releases
+        - These result in 4 bug fix releases
+        - For right now, USGS would like to stay with doing 4 support sprints each year
+    - Do we stick with 4 feature releases a year?
+      - This is the momentum carried forward approach
+      - Keeping release candidates is good
+        - Can we swap a feature release for a longer release candidate for the LTS version?
+        - There is some amount of support in the community for trying out release candidates
+        - Is 3 months enough time?
+        - Installation and upgrade process is a hinderance to people testing RCs and switching between versions
+        - Broader messaging around the annual RC would help drive engagement
+        - USGS can schedule support work between RC and full release to help support people testing it
 
 - Issue lifecycle
   - Comments from Jay [ISIS #3951](https://github.com/USGS-Astrogeology/ISIS3/issues/3951)
-
+  - Disussion on the issue has been good. TC doesn't need to get involved.
+  - If a contributor decides the TC needs to get involved they should open a PR to add this to the next meeting agenda and it will be discussed.
 
 ## Discussions for next meeting
-- 1 year LTS model
-  - Specific implementation details!
-- Issue lifecycle
-  - Comments from Jay [ISIS #3951](https://github.com/USGS-Astrogeology/ISIS3/issues/3951)
-  - We probably won't get to this until September
+- 1 year LTS
+  - USGS Astrogeology dev input
+  - Feature release cadence
+  - Bug fix release cadence
+  - Final plan and implementation schedule
+    - Committee to write this?
 
 ## Action Items
--
+- Jesse will review the contributing document and start an issue with what needs to be updated on it
```

Comment:
This was partially discussed during the meeting. Out contributing doc needs a once over. I am going to go through it and start some issues to update things and hopefully spark some policy discussion/codification.